src/resources/resources_html.js

```diff
@@ -0,0 +1,35 @@
+pc.extend(pc, function () {
+    'use strict';
+
+    var HtmlHandler = function () {
+
+    };
+
+    HtmlHandler.prototype = {
+        load: function (url, callback) {
+            pc.net.http.get(url, function (response) {
+                callback(null, response);
+            }, {
+                error: function (status, xhr, e) {
+                    callback(pc.string.format("Error loading html resource: {0} [{1}]", url, status));
+                }
+            });
+        },
+
+        open: function (url, data) {
+            return data;
+        },
+
+        patch: function (asset, assets) {
+        }
+    };
+
+    var createHtml = function (htmlString) {
+        return document.createRange().createContextualFragment(htmlString);
```

Comment:
Seems like well supported: https://developer.mozilla.org/en-US/docs/Web/API/Range/createContextualFragment
It takes valid html string, and converts it to abstract document fragment, which then can be appended as child to DOM.